src/pipecat/audio/filters/krisp_filter.py

```diff
@@ -20,6 +20,24 @@
     raise Exception(f"Missing module: {e}")
 
 
+class KrispProcessorManager:
+    """
+    Ensures that only one KrispAudioProcessor instance exists for the entire program.
+    """
+    _krisp_instance = None
+
+    @classmethod
+    def get_processor(cls, sample_rate: int, sample_type: str, channels: int, model_path: str):
```

Comment:
This is a workaround for cases where we know Krisp is always used with the same sample_rate, sample_type, and model_path. Further investigation is needed to fix the issue and release a new version of pipecat-krisp.